Source/options.js

```diff
@@ -512,6 +512,16 @@ $(function() {
     save('ngramMax', 'Maximum word sequence length saved');
   }
 
+  function validate_yandexTranslatorApiKey(){
+    var yandexKey = e("yandexTranslatorApiKey").value
+    if (yandexKey.match(/Unblock|Block/)){
+      alert("Please enter valid Yandex key.\nRemove Block key / Unblock key from the end of the key")
```

Comment:
After the alert, resetting the Yandex Key input field would be better. That would prevent the user from accidentally submitting an invalid key. Just like you reset password if it is incorrect.

Also I would suggest using the `status` function instead of `alert`. Many browsers block popups at first. 

```javascript
status('Invalid Key', 9000, 600, 'error');
```